lib/hermann/consumer.rb

```diff
@@ -24,6 +24,10 @@ class Consumer
     #
     def initialize(topic, opts = {})
```

Comment:
Will you update the RDoc to include the :`offset` option documentation including mention of `:start` and `:end`